README.md

```diff
@@ -1,6 +1,17 @@
 # code-reviewer
 
-Code Reviewer Application
+Anonymous online peer code review made easy.
+
+## Requirements
+
+MongoDB v4.2+ https://docs.mongodb.com/manual/installation/
+Node v10.16.3+ https://nodejs.org/en/download/
+Yarn v1.19.0+ https://yarnpkg.com/lang/en/docs/install
+
+## Getting Started (first time)
+
+run `mongod` to start MongoDB server at default port 27017
+run `yarn` in project root to install all node dependencies
```

Comment:
Format issue. I would write a commented bash command here. Something like:
```bash
$ mongod # Start MongoDB server...
```
I would remove the port too as that could change.